samples/tasks/scenarios/ovn-sandbox/start_sandbox.json

```diff
@@ -0,0 +1,21 @@
+{
+    "version": 2,
+    "title": "Start sandbox",
+    "subtasks": [{
+        "title": "Start sandbox on farm 0",
+        "workloads": [{
+            "name": "OvnSandbox.start_sandbox",
+            "args": {
+                "sandbox_start_args": {}
```

Comment:
Just done it.

For creating two farm nodes, deployment config file is:
<pre><code>
 {
    "type": "OvnMultihostEngine",
    "controller": {
        "type": "OvnSandboxControllerEngine",
        "deployment_name": "ovn-controller-node",
        "ovs_repo": "https://github.com/openvswitch/ovs.git",
        "ovs_branch": "master",
        "ovs_user": "rally",
        "net_dev": "eth1",
        "controller_cidr": "192.168.10.10/16",
        "provider": {
            "type": "OvsSandboxProvider",
            "credentials": [{ "host": "192.168.20.10", "user": "root"} ]
        }
    },
    "nodes": [
        {
            "type": "OvnSandboxFarmEngine",
            "deployment_name": "ovn-farm-node-0",
            "ovs_repo" : "https://github.com/openvswitch/ovs.git",
            "ovs_branch" : "master",
            "ovs_user" : "rally",
            "provider": {
                "type": "OvsSandboxProvider",
                "credentials": [ { "host": "192.168.20.20", "user": "root"}]
            }
        },
        {
            "type": "OvnSandboxFarmEngine",
            "deployment_name": "ovn-farm-node-1",
            "ovs_repo" : "https://github.com/openvswitch/ovs.git",
            "ovs_branch" : "master",
            "ovs_user" : "rally",
            "provider": {
                "type": "OvsSandboxProvider",
                "credentials": [{"host": "192.168.20.30","user": "root"}]
            }
        }
    ]
}
</code></pre>
You can add more farm nodes under 'nodes'.


And create_sandbox.json can be: 
<pre><code>
{% set farm = farm or "ovn-sandbox-node-0" %}
{% set amount = amount or 50 %}
{% set start_cidr = start_cidr or "192.168.64.0/16" %}
{% set tor = tor or "" %}
{
    "version": 2,
    "title": "Create sandbox",
    "subtasks": [{
        "title": "Create sandboxes",
        "workloads": [{
            "name": "OvnSandbox.create_sandbox",
            "args": {
                "sandbox_create_args": {
                    "farm": "{{farm}}",
                    "amount": {{amount}},
                    "batch": 50,    
                    "start_cidr": "{{start_cidr}}",
                    "net_dev": "eth0",
                    "tag": "{{tor}}"
                }
            },
            "runner": { "type": "serial", "times": 1 },
            "context": {
               "ovn_multihost" : {"controller": "ovn-controller-node"} 
            }
        }]
    }]
}
</code></pre>

Then you can use shell to create sandboxes on both farm nodes:
<pre><code>
$ rally-ovs task start create_sandbox.json --task-args '{"farm":"ovn-sandbox-node-0",  "start_cidr":"192.168.1.0/16",   "tor":"ToR-01"}'
$ rally-ovs task start create_sandbox.json --task-args '{"farm":"ovn-sandbox-node-1",  "start_cidr":"192.168.3.0/16",   "tor":"ToR-11"}'
</code></pre>